test/helpers/envtest.go

```diff
@@ -69,6 +71,7 @@ func init() {
 			filepath.Join(root, "config", "crd", "bases"),
 			filepath.Join(root, "controlplane", "kubeadm", "config", "crd", "bases"),
 			filepath.Join(root, "bootstrap", "kubeadm", "config", "crd", "bases"),
+			filepath.Join(root, "test", "infrastructure", "docker", "config", "crd", "bases"),
```

Comment:
Why are we using DockerMachine instead of `TestGenericInfrastructureCRD`?

Reply:
Don't we need the Kinds to already exist to use envtest?

Reply:
Yeah, the types for the fake CRDs are specified a few lines down, they're free-form, so you can use unstructured objects to manage them, they can have any field you'd like